wearos_app/build.gradle.kts

```
@@ -0,0 +1,73 @@
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+    id("kotlin-kapt")
+    id("kotlin-android-extensions")
+}
+
+android {
+    compileSdkVersion(Config.Android.compileSdk)
+
+    defaultConfig {
+        applicationId = "io.homeassistant.companion.wearos"
+        minSdkVersion(Config.Android.minSdkWear)
+        targetSdkVersion(Config.Android.targetSdk)
+        versionCode = 1
+        versionName = "1.0"
```

We should use the same versioning as the android app:
```suggestion
        versionCode = System.getenv("VERSION_CODE")?.toIntOrNull() ?: 1
        versionName = System.getenv("VERSION") ?: "LOCAL"
```